.circleci/config.yml

```diff
@@ -49,6 +49,7 @@ aliases:
       branches:
         only:
           - main
+          - paywalls
```

Comment:
`paywalls` is sort of a "main" branch, we can have this for now until it's merged to catch any issues early.